src/test/java/dev/jbang/source/TestModule.java

```diff
@@ -43,6 +43,15 @@ public class TestModule extends BaseTest {
 			"    }\n" +
 			"}\n";
 
+	String srcWithVersionRangeDep = "//MODULE testmodule\n" +
+			"//DEPS info.picocli:picocli:4.6+\n" +
```

Comment:
added this test as realized the module support seem to rely on *exact* string matches rather than understanding of resolved dependencies - is that a known/understood limitation @quintesse ?

Reply:
looks like Module generation should only compare on the group + artifict + type ..ignore version range or any properties?

Reply:
I'm not sure what you're referring to @maxandersen . What exact string matches? What understanding of resolved dependencies? There's not really a relationship between modules and maven dependencies implemented (at least as far as I remember).

Reply:
it's https://github.com/jbangdev/jbang/pull/2135